lib/eventboss/error_handlers/sentry.rb

```diff
@@ -0,0 +1,19 @@
+module Eventboss
+  module ErrorHandlers
+    class Sentry
+      def call(exception, context = {})
+        eventboss_context = { component: 'eventboss' }
+        eventboss_context[:action] = context[:processor].class.to_s if context[:processor]
+
+        default_options = { }
+
+        ::Sentry.with_scope do |scope|
+          scope.set_tags(
+            context.merge(eventboss_context, default_options)
+          )
```

Comment:
@AdamMarkowski  What do you think about segregating the event data this way:
```suggestion
          scope.set_tags(eventboss_context)
          scope.set_context('context', context)
          )
```
[Tags](https://docs.sentry.io/platforms/ruby/enriching-events/tags/) - As they are both indexed we will be able to filter and search Sentry issues by the tags.
[Context](https://docs.sentry.io/platforms/ruby/enriching-events/context/) - We cannot search them, but they are viewable on the Sentry issue page.

Reply:
I think that tags fits here, as they are [:worker_id](https://github.com/AirHelp/eventboss/blob/cf6a1ae700503981f887de1bd03e6077ace2a5e0/lib/eventboss/worker.rb#L29), [:pooler_id](https://github.com/AirHelp/eventboss/blob/bc69db8820ab9ddaa7f9100cf27047a8636bd8d9/lib/eventboss/long_poller.rb#L62) and [:processor](https://github.com/AirHelp/eventboss/blob/c911b76f3447f806588310d11db399fe55f3897e/lib/eventboss/unit_of_work.rb#L23)

Reply:
@nnnande I chose `set_tags` here because we have the ability to search through them, as opposed to context where we cannot - is it ok for you?

Reply:
Sure, sounds reasonable - being able to filter stuff by tags will help with no doubts @AdamMarkowski